lib/oli_web/live/community_live/show.ex

```diff
@@ -23,37 +31,43 @@ defmodule OliWeb.CommunityLive.Show do
 
   def mount(%{"community_id" => community_id}, _session, socket) do
     socket =
-      case Groups.get_community(community_id) do
+      with %Community{status: status} = community <- Groups.get_community(community_id),
+           "active" <- Atom.to_string(status) do
+        changeset = Groups.change_community(community)
+
+        assign(socket,
+          community: community,
+          changeset: changeset,
+          breadcrumbs: breadcrumb(community_id)
+        )
+      else
         nil ->
           socket
           |> put_flash(:info, "That community does not exist.")
           |> push_redirect(to: Routes.live_path(OliWeb.Endpoint, Index))
 
-        community ->
-          changeset = Groups.change_community(community)
-
-          assign(socket,
-            community: community,
-            changeset: changeset,
-            breadcrumbs: breadcrumb(community_id)
-          )
+        status ->
```

Comment:
I didn't spend the time to look at this fully - but is there something to match on such as {:error, status}?

Reply:
I explained it in [this comment](https://github.com/wyeworks/oli-torus/pull/5#discussion_r749416374), and suggested an alternative there